packages/react/src/components/Button/Button.mdx

```diff
@@ -2,7 +2,7 @@ import { Story, Props, Source, Preview } from '@storybook/addon-docs/blocks';
 import { withKnobs, boolean, select, text } from '@storybook/addon-knobs';
 import Button from '../Button';
 import ButtonSet from '../ButtonSet';
-import { Add16, Delete16 } from '@carbon/icons-react';
+import { Add, TrashCan } from '@carbon/react/icons';
```

Comment:
```suggestion
import { Add, TrashCan } from '@carbon/icons-react';
```

I think we'll need this to stay `icons-react` on our end, unfortunately, since technically `@carbon/react` shouldn't be defined at this level (since we're authoring the library)

I think this will work in storybook if it looks upwards for our local package.json but may not always work out, potentially.

Reply:
Ohh interesting. Yeah it works now but I didn't even think about it. Might be a webpack local resolution feature or something? Either way I agree and would rather not rely on it.